source/SpinalHDL/Libraries/EDA/altera/quartus_flow.rst

```diff
@@ -0,0 +1,58 @@
+
+QuartusFlow
+===========
+
+Introduction
+------------
+
+A compilation flow is an Altera-defined sequence of commands that use a combination of command-line executables.
+A full compilation flow launches all Compiler modules in sequence to synthesize, fit, analyze final timing, and generate a device programming file. 
+Tools in `this file <https://github.com/SpinalHDL/SpinalHDL/blob/dev/lib/src/main/scala/spinal/lib/eda/altera/QuartusFlow.scala>`_ help you get rid of redundant Quartus GUI.
+
+For a single rtl file
+---------------------
+
+The object ``spinal.lib.eda.altera.QuartusFlow`` can automatically report the area and maximum frequence of a single rtl file.
+
+Example
+^^^^^^^
+
+.. code-block:: scala
+
+   val report = QuartusFlow(
+      quartusPath="/eda/intelFPGA_lite/17.0/quartus/bin/",
+      workspacePath="/home/spinalvm/tmp",
+      toplevelPath="TopLevel.vhd",
+      family="Cyclone V",
+      device="5CSEMA5F31C6",
+      frequencyTarget = 1 MHz
+   )
+   println(report)
+
+The code above will create a new Quartus project with ``TopLevel.vhd``. By the way, if you use the device ``EP4CE6E22C8N``, the family should be ``Cyclone IV E``.
```

Comment:
It's fine as is (IMHO).. but I think in the feature is better to have a table with family and most used device serial.

Reply:
maybe there can link to altera website?

Reply:
I haven't found any 😞

Reply:
The `QuartusFlow` don't use the `family` or `device`. These two strings are directly passed to Quartus command line.

Reply:
Maybe we need just a note like this:
```suggestion
The code above will create a new Quartus project with ``TopLevel.vhd``.

.. note::
   The ``family`` and ``device`` values are passed straight to the Quartus CLI as parameters. Please check the Quartus documentation for the correct value to use in your project.
```

what do you think?

Reply:
Fantastic！I'm learning a lot English from you. 😂